src/OverrideFileService.cpp

```diff
@@ -0,0 +1,33 @@
+#include "OverrideFileService.h"
+
+#include <QMessageBox>
+
+namespace BsaPacker {
+	const uint16_t FALLOUT_3_NEXUS_ID = 120;
+	const uint16_t NEW_VEGAS_NEXUS_ID = 130;
+
+	OverrideFileService::OverrideFileService(
+		const IFileWriterService* fileWriterService)
+		: m_FileWriterService(fileWriterService)
+	{
+	}
+
+	// TODO: Add detection for Command Extender and JIP LN NVSE and warn if missing
+	bool OverrideFileService::CreateOverrideFile(const int nexusId,
+		const QString& modPath,
+		const QString& archiveNameBase) const {
+
+		if (nexusId != FALLOUT_3_NEXUS_ID && nexusId != NEW_VEGAS_NEXUS_ID) {
+			return false;
+		}
+
+		if (QMessageBox::question(nullptr, "BSAPacker", "Create .override file?",
+			QMessageBox::Yes | QMessageBox::No) & QMessageBox::No) {
+			return false;
+		}
```

Comment:
Use `questionboxmemory` instead of `QMessageBox` directly. It allows users to check a "Remember Selection" checkbox and skip the dialog the next time. 
https://github.com/ModOrganizer2/modorganizer-uibase/blob/master/src/questionboxmemory.h

You can find examples of usages in the main repo, e.g.: https://github.com/ModOrganizer2/modorganizer/blob/5adb1ec43d476a8733fdf79274c44d21046e02da/src/profile.cpp#L860-L872

Reply:
Thanks, I have changed it to use QuestionBoxMemory.